docs/diagrams/SetThemeSequenceDiagram.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+!include style.puml
+
+box Ui UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+
+box Ui
+participant "primaryStage:Stage" as Stage #darkgrey
+
+
+box Ui LOGIC_COLOR_T1
+participant ":Scene" as Scene LOGIC_COLOR
+end box
```

Comment:
I think it would be better if you just use 1 box for all three participants since they are all in the same Ui component. Adding these different colors might make our diagrams inconsistent.